extensions/objects/templates/unicode-string-editor.component.spec.ts

```diff
@@ -94,9 +96,14 @@ describe('UnicodeStringEditorComponent', () => {
     component.ngOnInit();
     spyOn(component, 'closeEditor').and.callThrough();
     spyOn(component, 'replaceValue').and.callThrough();
+    // After execution of component.ngOnInit();, component.active is set to
+    // false. To test the subscribed observable, it needs to be set to true
+    // to make the code inside the run.
```

Comment:
What does this last part mean ("to make the code inside the run")? It doesn't scan.